.gitignore

```diff
@@ -64,3 +64,8 @@ ipch/
 CMakeLists.txt
 compile_commands.json
 /.gitmodules
+
+# IDE-related files for https://github.com/samrocketman/endless-sky-vscode-devcontainer
+/.clang-format
+*.AppImage
```

Comment:
Are there multiple AppImages being produced? Also this goes into the binary section.

Reply:
Yes, per commit.  Here's an example of my current commit assuming I do not clean the workspace.

```
$ ls -1 *.AppImage
Endless_Sky-92d0f94b-x86_64.AppImage
Endless_Sky-e73d4ca8-x86_64.AppImage
Endless_Sky-f62002c3-x86_64.AppImage
```

Reply:
Added to binary section.